lessons/module-2/objects-and-prototype-chain.md

```diff
@@ -96,7 +96,19 @@ Instructor.prototype.gradeProjects = function(project) {
 let instructor = new Instructor('Robbie', 'POST Requests');
 ```
 
-In order to add methods to our template, we had to manipulate the `prototype` directly.  This is exactly what our classes do under the hood.
+In order to add methods to our template, we had to manipulate the `prototype` directly. We can do with with our `Array` type. First, we will define a new prototype method on `Array`, and then access that method like we would for `.map()` or `.filter()`.
+
+```js
+Array.prototype.getLast = function() {
+  return this[this.length - 1]
+}
+
+const ingredients = ['sauce', 'cheese', 'crust', 'peppers']
+
+ingredients.getLast()
+```
+
+What is `this` referring to in the example above?
```

Comment:
Very good example - it's simple enough to keep their focus on the prototype chain instead of the logic of the `getLast` function itself